src/components/camera/client.ts

```diff
@@ -59,6 +82,32 @@ export class CameraClient implements Camera {
     return resp.image;
   }
 
+  async getImages(
+    filterSourceNames: string[] = [],
+    extra = {},
+    callOptions = this.callOptions
+  ) {
+    const request = new GetImagesRequest({
+      name: this.name,
+      filterSourceNames,
+      extra: Struct.fromJson(extra),
+    });
+
+    this.options.requestLogger?.(request);
+
+    const resp = await this.client.getImages(request, callOptions);
+    const images = resp.images.map((image) => ({
+      sourceName: image.sourceName,
+      image: image.image,
+      mimeType: image.mimeType || formatToMimeType(image.format),
+    }));
+    const metadata: ResponseMetadata = {
+      capturedAt: resp.responseMetadata?.capturedAt ?? new Timestamp(),
+    };
+
+    return [images, metadata] as [NamedImage[], ResponseMetadata];
```

Comment:
```suggestion
    return { images, metadata } 
```
this much more standard than returning an array tuple